File: pythonassignment/Ditel 3/palindromes.py
number = int(input('Enter a 5 digit number: '))

originalNumber = number
reverse = 0

while number != 0:
	newNumber = number % 10
	reverse = reverse * 10 + newNumber
	number /= 10

	if originalNumber == reverse:
		print('is palindrome')
	

File: java assignment/ditel 3/Palindromes.java
import java.util.Scanner;
public class Palindromes{
	public static void main(String [] args){

		Scanner collect = new Scanner(System.in);

		System.out.println("Enter a 5digit number: ");
		int number = collect.nextInt();

		int originalNumber = number;
		int reverse = 0;

		while(number != 0){
			int newNumber = number % 10;
			reverse = reverse * 10 + newNumber;
			number /= 10;
		}
		 if(originalNumber == reverse){
			System.out.println("its palindrome");
		}
	
	}
}